### PR TITLE
VS Code 1.35.0 Extension API

### PR DESCRIPTION
**Pull request by rebornix**
Updating VS Code Extension API. See https://github.com/microsoft/vscode/issues/70175 for details

Changes in this PR

* Added `Comments` API
* Updated js docs.

### CODE REVIEW

**Review by kieferrm — APPROVED** [review]

File: types/vscode/index.d.ts
Comment (amcasey) [inline types/vscode/index.d.ts]: @kieferrm, if you list individual owners (e.g. yourself) in Definitions By, the PR will get fast-tracked as owner-approved.  Of course, that will be moot once @sandersn confirms you have merge permission. :smile:

**Pull request merged**
